### ISSUE TITLE
Feature Request: Stock Tweens for sound, particle, material

### ISSUE THREAD

**Issue by Ali-RS** [jMonkeyEngine-Contributions/Lemur#69]
Would be very useful to have these Tweens :

- SoundTween: play a sound
- EffectTween : fire a particle emitter
- MaterialTween: animate material’s properties
- ChangeActionTween: tells the AnimComposer to change the current Action

**Comment by pspeed42** [comment]
Yeah, the thing is that for 'instant' things like this you can just use the call method action almost as easily as a separate tween.  So I'm still trying to figure out what all value those sorts of things should add over callMethod() and callTweenMethod().

Material parameters (or even better matparam overrides for spatials) might be nice, though.

**Comment by mrlem** [comment]
For a SoundTween, the interest I could see is the sound duration handling: you could create a sequence with some action *after* the sound is done playing (not merely after it starts playing).
